src/averageAges.js

```diff
@@ -38,6 +52,30 @@ function calculateMenAverageAge(people, century) {
  */
 function calculateWomenAverageAge(people, withChildren) {
   // write code here
+  const children = people.filter(child =>
+    child.mother !== null);
+  let findWomen = people.filter(listOfWomen =>
+    listOfWomen.sex === 'f');
+
+  if (arguments.length > 1) {
+    const listWithChildren = [];
+
+    for (let i = 0; i < children.length; i++) {
```

Comment:
// avoid using loop and forEach
  // replace `if ()` statement with &&, || or ?:

Reply:
you can definetely make it easier, all you need to find is if there are any children whose `mother` value is equal to any `woman.name`